src/status_im/contexts/chat/messenger/composer/gesture.cljs

```diff
@@ -68,7 +68,6 @@
                             (if-not keyboard-shown
                               (do ; focus and end
                                 (when (< (oops/oget event "velocityY") constants/velocity-threshold)
-                                  (reanimated/set-shared-value container-opacity 1)
```

Comment:
Also here, is the container's opacity being handled somewhere else on-focus?

Reply:
yes